CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ New features:
 Bugfixes:
 
 Other improvements:
+- `split` (#165)
```

Comment:
Could you follow the formatting used in the rest of this changelog (e.g. including your username)?

If you want to provide more details, do something like this
```
- main entry (#PR by @<name>)

  Some more details here
  
- next main entry...
```